sources/packages/backend/apps/db-migrations/src/sql/User/Create-user-history-table.sql

```diff
@@ -0,0 +1,61 @@
+-- Table to keep historical data.
+-- Columns history_timestamp and history_operation should be default columns for all the history tables.
+-- Columns other than the above-mentioned should reflect the columns from the original table without constraints and relationships.
+-- The order of the columns below follow the exact same order from the existing table.
+CREATE TABLE sims.users_history (
+  history_timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
+  history_operation VARCHAR(20) NOT NULL,
+  id INT,
+  user_name VARCHAR(300),
+  email VARCHAR(300),
+  first_name VARCHAR(100),
+  last_name VARCHAR(100),
+  created_at TIMESTAMP WITH TIME ZONE,
+  updated_at TIMESTAMP WITH TIME ZONE,
+  is_active BOOLEAN,
+  creator INT,
+  modifier INT,
+  identity_provider_type sims."identity_provider_types"
+);
+
+CREATE INDEX users_history_timestamp ON sims.users_history(history_timestamp);
+
+COMMENT ON INDEX sims.users_history_timestamp IS 'Historical data index to improve point-in-time data retrieval.';
+
+COMMENT ON TABLE sims.users_history IS 'Historical data generated after records are inserted or updated.';
+
+COMMENT ON COLUMN sims.users_history.history_timestamp IS 'Date and time the history record was inserted.';
+
+COMMENT ON COLUMN sims.users_history.history_operation IS 'Database operation that generates the history.';
+
+COMMENT ON COLUMN sims.users_history.id IS 'Historical data from the original table. See original table comments for details.';
+
+COMMENT ON COLUMN sims.users_history.user_name IS 'Historical data from the original table. See original table comments for details.';
+
+COMMENT ON COLUMN sims.users_history.email IS 'Historical data from the original table. See original table comments for details.';
```

Comment:
Should we ignore adding comments for original table copy columns instead of using dummy comment ?

Reply:
I see the point, I had the same doubt and ended up adding the comments in a way that they would point to the real business comments to keep the idea of always adding a comment to the DB objects.
I would be ok either way in this situation but I preferred the approach in place right now.